src/core/layout/qgslayoutitemattributetable.cpp

```diff
@@ -628,16 +634,6 @@ bool QgsLayoutItemAttributeTable::getTableContents( QgsLayoutTableContents &cont
     ++counter;
   }
 
-  //sort the list, starting with the last attribute
-  QgsLayoutAttributeTableCompare c;
```

Comment:
You can completely remove QgsLayoutAttributeTableCompare now -- it's private and unused.

Reply:
The full class?

Reply:
Done